projects/packages/account-protection/src/css/strength-meter.css

```
@@ -0,0 +1,96 @@
+.validation-checklist {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+    margin: 16px 0;
+}
+
+.validation-item {
+    display: flex;
+    align-items: center;
+    gap: 8px;
+    margin-bottom: 0;
+
+    .validation-icon {
+        height: 24px;
+    }
+    
+    .validation-message {
+        margin-top: 0;
+    }
+}
+
+.info-popover {
+    position: relative;
+    display: inline-block;
+    height: 20px;
+}
+
+.info-icon {
+    height: 20px;
+    cursor: pointer;
+}
+
+.popover {
+    display: none;
+    position: absolute;
+    bottom: 30px;
+    left: 50%;
+    transform: translateX(-50%);
+    background: #333;
+    color: #fff;
+    padding: 6px 10px;
+    border-radius: 4px;
+    white-space: normal;
+    width: 150px;
+    font-size: 12px;
+    box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.1);
+    z-index: 10;
+    text-align: center;
+}
+
+.popover-arrow {
+    position: absolute;
+    bottom: -6px;
+    left: 50%;
+    transform: translateX(-50%);
+    border-left: 6px solid transparent;
+    border-right: 6px solid transparent;
+    border-top: 6px solid #333;
+}
+
+.strength-meter {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 30px;
+    padding: 0 16px;
+    margin-bottom: 16px;
+    border-radius: 0 0 4px 4px;
+    background-color: #8C8F94;
+}
+
+.strength-meter .strength {
+    display: flex;
+    align-items: center;
+    font-size: 12px;
+    font-weight: bold;
+    color: #1D2327;
+    margin: 0;
```

```suggestion
    margin: 0 1px;
```
Core forms have a 1px horizontal margin on inputs, matching that here will line up the strength meter with the password input width.

---

IIRC the margins vary from one form to the next. The only way to standardize things and ensure the input and strength meters remained the same width across all forms without applying conditional CSS was to remove the margins from all.

---

We could use a custom selector here, i.e.

```css
#profile-page .strength-meter {
    margin: 0 1px;
}
```

---

That seems it could do the trick, we'll also need a selector for the `user-new.php` form. Looks like we could use the form element id's for each `#your-profile` applies to current user (`profile.php`) and other user (`edit-user.php`) edits, `#createuser` applies to user creation (`user-new.php`)